src/coreclr/jit/emitxarch.h

```diff
@@ -786,111 +851,144 @@ void emitIns_R_AX(instruction ins, emitAttr attr, regNumber ireg, regNumber reg,
 
 void emitIns_AX_R(instruction ins, emitAttr attr, regNumber ireg, regNumber reg, unsigned mul, int disp);
 
-void emitIns_SIMD_R_R_I(instruction ins, emitAttr attr, regNumber targetReg, regNumber op1Reg, int ival);
+void emitIns_SIMD_R_R_I(
```

Comment:
As described in https://github.com/dotnet/runtime/pull/101886#issuecomment-2099045553, the main consideration is that `emitIns_SIMD_*` very explicitly needs to pass down `insOpts` to ensure that embedded broadcast/masking/rounding was handled. 

If you aren't passing it down, then you have potentially missed that handling and could end up with codegen bugs. So making it non-defaultable ensures that all code paths are explicitly considering this scenario and passing the computed options down otherwise (helping ensure we haven't missed any handling).